yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java

```diff
@@ -39,6 +39,9 @@ public class SdkConfig {
     @JsonProperty("allow_handoff")
     private final Boolean allowHandoff;
 
+    @JsonProperty("attempts_configuration")
```

Comment:
could u create a Property class in here as well?

Reply:
Done